test/async/retry_test.dart

```diff
@@ -0,0 +1,445 @@
+// Copyright 2014 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the 'License');
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an 'AS IS' BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+library quiver.async.retry_test;
+
+import 'dart:async';
+import 'dart:math' as math;
+
+import 'package:quiver/async.dart' as retry;
+import 'package:unittest/unittest.dart';
+
+main() {
```

Comment:
Should I make this conversion before this initial pull request is complete, or can it wait until a subsequent one?